src/Exception/ResponseExceptionFactory.php

```diff
@@ -0,0 +1,42 @@
+<?php
+
+namespace Jedkirby\ValPal\Exception;
+
+class ResponseExceptionFactory
+{
+    /**
+     * @var string
+     */
+    const GENERIC_ERROR = 'Unable to process valuation';
+
+    /**
+     * An array of ValPal specific error response codes and messages.
+     *
+     * @var array
+     */
+    public static $messages = [
```

Comment:
I actually use this array within tests to build a data provider (https://github.com/jedkirby/valpal-php/blob/feature/package/tests/ClientTest.php#L75) - Could either copy this array, or, create a getter for it?